dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -6,7 +6,7 @@ class Api::V1::SectionsController < Api::V1::JsonApiController
   # GET /api/v1/sections
   # Get the set of sections owned by the current user
   def index
-    render json: @sections.map(&:summarize)
+    render json: current_user.sections.map(&:summarize)
```

Comment:
I agree it'd be worth exploring limiting the `index` permission for admins in CanCan as a follow up (or tech debt item).